### ISSUE TITLE
Currency Change

### ISSUE THREAD

**Issue by Webstream2411** [amirhamza05/Student-Management-System#64]
Can u please tell me how to change currency displayed TK to INR

**Comment by Ucgodman** [comment]
HI, MY NAME IS UCGODMAN I FOUND OUT THIS STUDENT MANAGEMENT SYSTEM BY AMIRHAMZA05 WHICH I LOVE TO SETUP IN MY SYSTEM. AFTER DOWNLOADING THE FILE INTO MY SYSTEM FOLLOWED THE NECESSARY PROCEDURE BY AMIRHAM, YET I CAN NOT LOGIN INTO THE SYSTEM. I BELIEVED YOU SUCCEEDED WHICH YOU ARE ASKING FOR CHANGE IN CURRENCY. I JUST WANT TO ASK IF YOU CAN ASSIST ME ON HOW TO WERE ABLE TO SETUP YOUR OWN ON YOUR LOCAL DRIVE. MY COMMENT IS THE ONE BEFORE YOURS.

I WILL BE GLAD IF REQUEST IS GRANTED.

THANKS.

**Comment by Webstream2411** [comment]
Yes help you but you habe to buy hosting from us. We will completely host you this scipt.

**Comment by Ucgodman** [comment]
ok, but i need to set it up on the offilne on my system to know, if it what i want. then we can talk about hosting.
thanks

**Comment by amirhamza05** [comment]
> Can u please tell me how to change currency displayed TK to INR

This version currency change system not available. Hopefully we implement it in future.

**Comment by Webstream2411** [comment]
> ok, but i need to set it up on the offilne on my system to know, if it what i want. then we can talk about hosting. thanks

No we host online only.
